schema/market/sellTicket.schema.ts

```diff
@@ -9,6 +9,13 @@ export const SellTicketFormSchema = object({
       (value, context) => {
         return Boolean(value && value >= context.parent.minResalePrice)
       }
+    )
+    .test(
+      "Lower than or equal to the max resale price",
+      "Ticket resale price should be lower than the maximum price set by the event organizer",
```

Comment:
Ticket resale price should not exceed the maximum price set by the event organizer